api/helpers/gamestate/moves/resolve/ace.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  friendlyName: 'Resolve Ace One-Off',
+
+  description: 'Returns new GameState resulting from resolving an ace, which scraps all points on the field, and all jacks attached to those point cards.',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      description: 'The latest GameState before the ace scraps all point cards and their attachments',
+      required: true,
+    },
```

Comment:
Notably, unlike the actual `execute()` helpers, the one-off resolutions only require the `currentState` without caring about who is resolving etc.